mtags/src/main/scala-3/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -365,6 +323,48 @@ object CaseKeywordCompletion:
         defnSymbols.getOrElse(semancticName, -1)
       }
 
+  def subclassesForType(tpe: Type)(using Context): List[Symbol] =
+    // Symbol must be a subclass of every `AndType`
+    def getAndTypes(tpe: Type, acc: List[Type]): List[Type] =
+      tpe match
+        case AndType(tp1, tp2) =>
+          getAndTypes(tp2, getAndTypes(tp1, acc))
+        case t => t :: acc
+
+    val andTypes = getAndTypes(tpe, List.empty)
+
+    def getParentTypes(tpe: Type, acc: List[Symbol]): List[Symbol] =
+      tpe match
+        case AndType(tp1, tp2) =>
+          getParentTypes(tp2, getParentTypes(tp1, acc))
+        case OrType(tp1, tp2) =>
+          getParentTypes(tp2, getParentTypes(tp1, acc))
+        case t =>
+          tpe.typeSymbol :: acc
+
+    def isEhaustiveMember(sym: Symbol): Boolean =
```

Comment:
```suggestion
    def isExhaustiveMember(sym: Symbol): Boolean =
```
if you maybe update the PR I would fix the typo. Let me know when to look at the full PR!